src/appleseed/renderer/modeling/shadergroup/shadergroup.cpp

```diff
@@ -444,12 +444,25 @@ void ShaderGroup::set_surface_area(
 
 const char* ShaderGroupFactory::get_model()
 {
-    return "shadergroup";
+    return "shader_group";
+}
+
+DictionaryArray ShaderGroupFactory::get_input_metadata()
+{
+    DictionaryArray metadata;
```

Comment:
You need to include `foundation/utility/api/specializedapiarrays.h` for `DictionaryArray`.